### ISSUE TITLE
Talkbalk identifies checked and unchecked subjects the same

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#598]
On the subject selection page during onboarding, a user can select and unselect subjects by double-tapping on the selection which either places or removes a tick in the check box for each subject. However, Talkback cannot identify whether a subject that has been “activated” by double-tapping is either selected or unselected, so a user will not know if they have made a selection, which may be confusing.

Rajat’s comments: It doesn't tell whether the subject has been selected or not. It should follow a similar structure on settings screen > language (checked/unchecked box) GitHub #569. Also, when clicked on select subjects again after closing the screen, it shows the checkboxes are still selected and doesn't show anything on that screen. GitHub #520.

1. Ensure that users will be informed when a selection has been made or removed, otherwise blind or low vision users will not know what (if any) selections have been made. 

3. Update the label for the checkbox from “tickbox” to “checkbox”, which may be a more familiar term. 

![GitHub11](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/3aa92bb7-ff15-4f69-bcf9-7db67bfbb008)
